subprojects/ce-login/celogin/include/CeLogin.h

```diff
@@ -142,34 +161,138 @@ struct CeLoginRc
         mComponent(0 == reasonParm ? 0 : componentParm), mReason(reasonParm)
     {}
 
-    operator uint16_t() const
+    inline operator uint16_t() const
     {
         // Format manually to allow for consistant behavior on Big vs Little
         // Endian systems
         return (uint8_t)mComponent << 8 | (uint8_t)mReason;
     }
 
+    inline bool isSuccess() const
+    {
+        return Success == (*this);
+    }
+
     uint8_t mComponent;
     uint8_t mReason;
 };
 
-CeLoginRc getServiceAuthorityV1(
+struct AcfUserFields
+{
+    AcfUserFields() { clear(); }
+
+    void clear()
+    {
+        memset(this, 0x00, sizeof(*this));
```

Comment:
This will throw the following error:warning: 
'void* memset(void*, int, size_t)' clearing an object of non-trivial type 'struct CeLogin::AcfUserFields'; use assignment or value-initialization instead [-Wclass-memaccess]
|   186 |         memset(this, 0x00, sizeof(*this));
OpenBMC build will fail on all warnings.

Reply:
The change to the clearing function has addressed this. This code not longer throws a warning.